test/common/3scale_smtp.go

```diff
@@ -477,6 +493,27 @@ func sendTestEmail(ctx *TestingContext, t TestingTB) {
 	}
 }
 
+func reset3ScaleSecret(ctx *TestingContext, t TestingTB) (string, error) {
```

Comment:
```suggestion
func reset3ScaleSecret(ctx *TestingContext, t TestingTB) error {
```

Reply:
Fair point. Removed the secret from returns